File: tasks/arrayMaxConsecutiveSum2.js
function arrayMaxConsecutiveSum2(inputArray) {
  var best = inputArray[0];
  for(var i = 0; i < inputArray.length; ++i) {
    var tmp = 0;
    for(var j = i; j < inputArray.length; ++j) {
      tmp += inputArray[j];
      best = Math.max(best, tmp);
    }
  }
  return best;
}

File: tasks/arrayMaxConsecutiveSum2.cpp
int arrayMaxConsecutiveSum2(std::vector<int> x) {
  int best = x[0];
  for(int i = 0; i < x.size(); ++i)
  {
    int tmp = 0;
    for(int j = i; j < x.size(); ++j)
    {
      tmp += x[j];
      if(tmp > best)
        best = tmp;
    }
  }
  return best;
}
